src/index.tsx

```diff
@@ -1,39 +1,35 @@
 import * as React from 'react';
 import { Provider } from 'react-redux';
 import { createRoot } from 'react-dom/client';
-import { ConnectedRouter } from 'connected-react-router';
+import { BrowserRouter } from 'react-router-dom';
 
 import { App } from './app';
 import { removeItems } from '@utilities';
 import { AuthActionType } from '@store/enums';
-import { history, configureStore } from '@store/index';
+import { configureStore } from '@store/index';
 
 export const store = configureStore();
 
-const root = createRoot(document.getElementById('app') || document.createElement('div'));
-const renderRoot = (app: JSX.Element): void => root.render(app);
-const router = (Application: any): JSX.Element => (
-	<Provider store={store}>
-		<ConnectedRouter history={history}>
-			<Application />
-		</ConnectedRouter>
-	</Provider>
-);
+const node: HTMLElement | null = document.getElementById('app') || document.createElement('div');
+const root = createRoot(node);
+
+const renderRoot = (Application: any): void => {
+	root.render(
+		<Provider store={store}>
+			<BrowserRouter>
+				<Application />
+			</BrowserRouter>
+		</Provider>
+	);
+};
 
 removeItems();
 
 store.dispatch({ type: AuthActionType.RESET_AUTH });
 
-renderRoot(router(App));
-
-if (module.hot) {
-	module.hot.accept();
-
-	// eslint-disable-next-line
-	renderRoot(router(require('./app').App));
-}
+renderRoot(App);
 
-if (process.env.NODE_ENV === 'production' && 'serviceWorker' in navigator) {
+if (import.meta.env.PROD && 'serviceWorker' in navigator) {
```

Comment:
Remove the service worker initialization if you already configured the Vite PWA plugin. The plugin should generate and initialize its own service worker.